src/Expression/Simplifier.purs

```diff
@@ -31,26 +26,101 @@ simplify (ExpressionBinary operation leftExpression rightExpression) =
 
 simplify expression = expression
 
+trimTimesNodes :: Expression -> Expression -> Maybe Expression
+trimTimesNodes (ExpressionLiteral leftValue) (ExpressionLiteral rightValue) =
```

Comment:
Why not simply replace with one literal holding the product of the two rationals?

Reply:
Yeah, this one can be removed.